scipy/signal/ltisys.py

```diff
@@ -1508,6 +1513,143 @@ def __repr__(self):
+                          np.dot(self.D, other))
+
+    def __rmul__(self, other):
+        """Pre-multiply a scalar or matrix (but not StateSpace)"""
+        if not self._check_binop_other(other) or isinstance(other, StateSpace):
+            return NotImplemented
+
+        # For pre-multiplication only the output gets scaled
+        return StateSpace(self.A,
+                          self.B,
+                          np.dot(other, self.C),
+                          np.dot(other, self.D))
+
+    def __neg__(self):
+        """Negate the system (equivalent to pre-multiplying by -1)."""
+        return StateSpace(self.A, self.B, -self.C, -self.D)
+
+    def __add__(self, other):
+        """
+        Add two systems
+
+        Adds two systems in the sense of frequency domain addition. That
+        means, given two systems E1(s) and E2(s), their addition,
+        H(s) = E1(s) + E2(s), means that applying H(s) to U(s)
+        is equivalent to applying E1(s) and E2(s) separately and adding up the
+        result.
+        """
+        if not self._check_binop_other(other):
+            return NotImplemented
+
+        if isinstance(other, StateSpace):
+            # Disallow mix of discrete and continuous systems.
+            if type(other) is not type(self):
+                raise TypeError('Cannot add {} and {}'.format(type(self),
+                                                              type(other)))
+
+            if self.dt != other.dt:
+                raise TypeError('Cannot add systems with different `dt`.')
+            # Interconnection of systems
+            # x1' = A1 x1 + B1 u
+            # y1  = C1 x1 + D1 u
+            # x2' = A2 x2 + B2 u
+            # y2  = C2 x2 + D2 u
+            # y   = y1 + y2
+            #
+            # Plugging in yields
+            # [x1']   [A1 0 ] [x1]   [B1]
+            # [x2'] = [0  A2] [x2] + [B2] u
+            #                 [x1]
+            #  y    = [C1 C2] [x2] + [D1 + D2] u
+            a = linalg.block_diag(self.A, other.A)
+            b = np.vstack((self.B, other.B))
+            c = np.hstack((self.C, other.C))
+            d = self.D + other.D
+            return StateSpace(a, b, c, d)
+
+        other = np.atleast_2d(other)
+        if self.D.shape == other.shape:
+            # A scalar/matrix is really just a static system (A=0, B=0, C=0)
+            return StateSpace(self.A, self.B, self.C, self.D + other)
+        else:
+            raise ValueError('Other needs to have compatible dimensions.')
+
+    def __sub__(self, other):
+        if not self._check_binop_other(other):
+            return NotImplemented
+
+        return self.__add__(-other)
+
+    def __radd__(self, other):
+        if not self._check_binop_other(other):
+            return NotImplemented
+
+        return self.__add__(other)
+
+    def __rsub__(self, other):
+        if not self._check_binop_other(other):
+            return NotImplemented
+
+        return (-self).__add__(other)
+
```

Comment:
While we are at it and if you agree; Quickly scaling the model with `G/6.5` is also possible with `__truediv__()` method where you can simply test for float or int as you do for mult and then `return self*(1/other)`. You can also explicitly reject `__rtruediv__()` with a more meaningful error message.